### PR TITLE
Updated Tournaments

### PR DESCRIPTION
**Pull request by bvanderdrift**
* Added prizes for some tournaments
* Added rulings for tournaments
* Added some custom tournaments

Credits to @Exclaminator for the md versions of rules of the main tournaments

### CODE REVIEW

File: app/scripts/unofficial-tournaments.json [hunk 1]
```diff
@@ -92,7 +92,151 @@
...
+    "backgroundImagePath": "images/unofficial/achtung.jpg",
+    "sponsor": {
+      "name": "",
+      "imagePath": "images/unofficial/achtung.jpg",
+      "website": "http://curvefever.com/"
+    },
+    "prizes": [
+      "T.B.A."
```
Comment (skamoen) [inline app/scripts/unofficial-tournaments.json]: Are these unofficial tournament prizes actually to be announced or aren't there any prizes at all?

File: app/scripts/unofficial-tournaments.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
+    "description": "Minecraft is a sandbox video game created by Swedish programmer Markus 'Notch' Persson and later developed and published by Mojang. The creative and building aspects of Minecraft enable players to build constructions out of textured cubes in a 3D procedurally generated world. Other activities in the game include exploration, resource gathering, crafting, and combat. Multiple gameplay modes are available, including survival mode where the player must acquire resources to build the world and maintain health, a creative mode where players have unlimited resources to build with and the ability to fly, an adventure mode where players can play custom maps created by other players, and a spectator mode where players can fly around and clip through blocks, but cannot place or destroy any.\n\nThe Area FiftyLAN server is currently available at 37.187.73.200:27848",
```
Comment (skamoen) [inline app/scripts/unofficial-tournaments.json]: It's not a public server, so I would not put this on apublic website

File: app/scripts/unofficial-tournaments.json (hunk 1 above)
Comment (bvanderdrift) [inline app/scripts/unofficial-tournaments.json]: I changed prizes of tournaments without prizes to "none"

File: app/scripts/unofficial-tournaments.json (hunk 2 above)
Comment (bvanderdrift) [inline app/scripts/unofficial-tournaments.json]: Removed the URL

**Comment by bvanderdrift** [comment]
Please hold until we have the last rules implemented.

**Comment by bvanderdrift** [comment]
Everything is updated. @martijn9612 could you PR check?

**Comment by martijn9612** [comment]
There are conflicts in the RL - Technical issues part. The part about the wired connection and wireless is wierd.
```3. Technical Issues\n* If either player disconnects from the game due to technical reasons and the game is heavily favoured to the extent that the game would definitely have been won by one player, provided a screenshot of the game prior to the disconnect, an administrator may rule the game in favour of said player. Should the game be too close to call, it will be replayed.\n* Our main source for tournament internet will be wired internet. If a player uses an alternative connection they do so at their own risk and can’t be guaranteed the same stability. \n\n\n```

**Comment by martijn9612** [comment]
In TF2, bombomnicon ban is stated twice.

File: app/scripts/unofficial-tournaments.json [hunk 3]
```diff
@@ -107,6 +179,150 @@
...
+      "PC",
+      "Mac"
+    ],
+    "footer": false
+  },
+  {
+    "title": "MK",
+    "subtitle": "1 v 1",
+    "headerTitle": "Mario Kart: Double Dash!!",
+    "description": "Double Dash!! challenges Mario series player characters to race against each other on Mario-themed tracks. One player drives the kart, and the other uses items. Players can switch at any time. There are 20 characters to select from in total, each of which with a special item, and with eleven characters being new to the series.\n\n# Schedule\n## Friday:\nStart: 21:30. \nEnd: Depending on number of players.\n\n# Rules\n## 1 Format:\nElimination rounds: 4 players in split screen. First to win 2 races, proceeds to the next round.\n\nQuarter finales: 1v1 elimination. First player to win 2 races, proceed to semi finales.\n\nSemi finales: 1v1 elimination. First player to win 2 races, proceed to finale.\n\nFinale: 1v1 elimination. First player to win 2 races, wins.\n\n## Console\nMario Kart Double Dash will be played on a the Nintendo Gamecube. Consoles and controllers will be provide.",
```
Comment (martijn9612) [inline app/scripts/unofficial-tournaments.json]: Last word: provided

**Comment by martijn9612** [comment]
LGTM, all issues fixed!

**Pull request merged**